meta-ibm/recipes-phosphor/webui/webui-vue/changeLogo.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+model=($(busctl get-property xyz.openbmc_project.Inventory.Manager /xyz/openbmc_project/inventory/system/chassis/motherboard/bmc xyz.openbmc_project.Inventory.Decorator.Asset SubModel))
+echo ${model[1]}
```

Comment:
What does this  write out? Is it needed?

Reply:
Sorry, I refer to these files here
https://github.com/ibm-openbmc/openbmc/commit/69ae2e079a8643ace8cf1d44988174cc8f2cf8a9
Removed